inc/composer/class-command.php

```diff
@@ -320,18 +335,130 @@ protected function logs( InputInterface $input, OutputInterface $output ) {
 	protected function shell( InputInterface $input, OutputInterface $output ) {
 		$columns = exec( 'tput cols' );
 		$lines = exec( 'tput lines' );
+		$command_prefix = $this->get_base_command_prefix();
 		passthru( sprintf(
-			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash',
-			'vendor/altis/local-server/docker',
-			escapeshellarg( getcwd() ),
-			$this->get_project_subdomain(),
+			"$command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash",
 			$columns,
 			$lines
 		), $return_val );
 
 		return $return_val;
 	}
 
+	protected function db( InputInterface $input, OutputInterface $output ) {
+		$db = $input->getArgument( 'options' )[0] ?? '';
+		$columns = exec( 'tput cols' );
+		$lines = exec( 'tput lines' );
+
+		$base_command_prefix = $this->get_base_command_prefix();
+
+		$base_command = sprintf(
+			"$base_command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d db",
+			$columns,
+			$lines
+		);
+
+		$return_val = 0;
+
+		switch ( $db ) {
+			case 'info':
+				$connection_data = $this->get_db_connection_data();
+
+				$db_info = <<<EOT
+<info>Root password</info>:  ${connection_data['MYSQL_ROOT_PASSWORD']}
+
+<info>Database</info>:       ${connection_data['MYSQL_DATABASE']}
+<info>User</info>:           ${connection_data['MYSQL_USER']}
+<info>Password</info>:       ${connection_data['MYSQL_PASSWORD']}
+
+<comment>Version</comment>:        ${connection_data['MYSQL_VERSION']}
+<comment>Host</comment>:           ${connection_data['HOST']}
+<comment>Port</comment>:           ${connection_data['PORT']}
+
+EOT;
+				$output->write( $db_info );
+				break;
+			case 'spf':
```

Comment:
```suggestion
			case 'spf':
			case 'sequel':
```
Spoke to Ryan and we'll use `sequel` as the primary name for this command